### PR TITLE
Building Block "Toolbars" refactoring

### PR DESCRIPTION
**Pull request by pivanov**

### CODE REVIEW

**Comment by pivanov** [comment]
@fabi1cazenave and @basiclines you can review it :)

File: widgets/toolbars/index.html [hunk 1]
```diff
@@ -1,52 +1,51 @@
...
-          margin: -1px 0 0;
-          background-color: #f5f5f5;
-          padding: 0.4rem 0.4rem 0.4rem 3rem;
-          border: solid 1px #e8e8e8;
-        }
-    </style>
-    <title>Toolbars</title>
-    <meta name="description" content="Contains actions, indicators, and navigation associated with the current view">
-    <meta name="requires" content="none">
-    <meta name="target" content="html">
```
Comment (basiclines) [inline widgets/toolbars/index.html:34]: @pivanov we should kept the `<meta name="target" content="html">` in BB repo (not for gaia merge) for still having the "switch background" behavior

**Comment by basiclines** [comment]
Also i think that we must kept the structure `/images/icons/` for being synced with all the others BB :D
Thx for your work!

**Comment by pivanov** [comment]
fix them :) 10x for quick review :)

**Comment by basiclines** [comment]
Thank you @pivanov this looks much better! I will take a look at the rendered html later. But seems pretty solid

**Pull request merged**